common/Renderers/DevHomeChoiceSetWithDynamicRefresh.cs

```diff
@@ -88,6 +88,7 @@ private Grid SetupParentComboBoxForDynamicRefresh(AdaptiveChoiceSetInput choiceS
         // Setup event handlers
         comboBox.SelectionChanged += RefreshCardOnSelectionChanged;
         comboBox.Unloaded += RemoveEventHandler;
+        comboBox.Loaded += AddEventHandlers;
```

Comment:
We never unsubscribe from this event. Can SetupParentComboBoxForDynamicRefresh be called multiple times so we subscribe for comboBox.Loaded multiple times?

Reply:
Removed the AddEventHandlers and the RemoveEventHandler and instead used a single `WeakEventListener` to accomplish what they were doing just for the selectionChanged event. The only event I really needed to fire was the SelectionChanged for the combo box. The Loaded and Unloaded events were not needed.